jinahub/rankers/LightGBMRanker/lightgbm_ranker.py

```diff
@@ -14,22 +14,9 @@ class LightGBMRanker(Executor):
     """
     Computes a relevance score for each match using a pretrained learning-to-rank model trained with LightGBM (https://lightgbm.readthedocs.io/en/latest/index.html)
 
-    :param query_features: name of the features to extract from query Documents and used to compute relevance scores by the model loaded
-    from model_path
-    :param match_features: name of the features to extract from match Documents and used to compute relevance scores by the model loaded
-    from model_path
-    :param relevance_label: If call :meth:`train` endpoint, the label will be used as groundtruth for model training. If on :meth:`rank` endpoint, the
-    label will be used to assign a score to :attr:`Document.scores` field.
-    :param model_path: path to the pretrained model previously trained using LightGBM.
-    :param params: Parameters used to train the LightGBM learning-to-rank model.
-    :param categorical_query_features: name of features contained in `query_features` corresponding to categorical features.
-    :param categorical_match_features: name of features contained in `match_features` corresponding to categorical features.
-    :param query_features_before: True if `query_features` must be placed before the `match` ones in the `dataset` used for prediction.
-    :param args: Additional positional arguments
-    :param kwargs: Additional keyword arguments
     .. note::
-        The name of the features are used to extract the features from incoming `documents`. Check how these features are accessed in
-        :class:`Document` at https://docs.jina.ai/api/jina.types.document/
+        The name of the features used by this Executor are used to extract the features from incoming `documents`.
```

Comment:
Why is this here? It should be under `features` docstring, no?

Reply:
This applies for all the *_features parameters in the constructor.
If I put it in the constructor docstring, it will not be displayed.
I can't put it after the parameters because they will show up in one of the parameter's description.
I'll improve the docstringso that it's clear we're referring to the constructor parameters